linux_memory_usage.py

```diff
@@ -32,9 +37,9 @@ def main():
         'Mapped', 'Active', 'Inactive'
     ]
 
-    template = "servers.{0}.system.memory.{1} {2} {3}"
+    template = args.prefix + '.{1} {2} ' + str(int(time()))
```

Comment:
You are right, sry, I thought that you python will call `str(...)` by itself while concatenating. Then nevermind.